modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -620,11 +621,28 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
       userSigningMaterial.bitgoNShare,
       userSigningMaterial.backupNShare,
     ]);
+    const signingKeyWithChallenge = MPC.signChallenge(signingKey.xShare, signingKey.yShares[3]);
+    const bitgoChallenge = await getTxRequestChallenge(
+      this.bitgo,
+      this.wallet.id(),
+      txRequestId,
+      '0',
+      requestType,
+      'ecdsa'
+    );
 
     const threshold = 2;
     const numShares = 3;
     const uShares = Ecdsa.shamir.split(BigInt(userSigningMaterial.pShare.uu), threshold, numShares);
-    const userSignShare = await ECDSAMethods.createUserSignShare(signingKey.xShare, signingKey.yShares[3]);
```

Comment:
please rebase from master, seem like receive address work will be overwritten if this merges

Reply:
@johnoliverdriscoll had asked that we revert the derivation stuff until we can hash out a better plan.